tajo-core/src/main/java/org/apache/tajo/master/GlobalEngine.java

```diff
@@ -143,6 +148,18 @@ private QueryContext createQueryContext(Session session) {
       newQueryContext.putAll(CommonTestingUtil.getSessionVarsForTest());
     }
 
+    // Set queryCache in session
+    if (session.getQueryCache() == null) {
+      session.setQueryCache(CacheBuilder.newBuilder()
+        .maximumSize(200)
```

Comment:
That's a good idea. No problem. By the way, the weight should be estimated by SQL query string length, right? Or, do you have some Util to measure Expr memory consumption? Up to now, I can not find that kind of code in Expr. Do you think SQL query string length is okay?